src/components/common/SessionApprove.vue

```diff
@@ -89,6 +87,11 @@ function parseTx(signMessage) {
   }
 }
 
+function parseFee(message) {
```

Comment:
Is there a specific script file that parseTx and parseFee could be kept in? @faboweb

Reply:
Not yet, feel free to create one. Something like 'scripts/parsers.je:?